### ISSUE TITLE
Support for an `artboard` rule?

### ISSUE THREAD

**Issue by bomberstudios** [kvendrik/sketchlint#1]
First of all: congratulations for the work here. The tool looks amazing.

It would be nice to have a rule for `artboard` (and maybe even `group`?) so that a check could be made for the layers in it (think "contrast ratio check", and stuff like that)

Currently this can be done by checking if `layer` is an Artboard, but I think it'd make the code in the config file a bit nicer : )

**Comment by bomberstudios** [comment]
Just saw https://github.com/kvendrik/sketchlint/compare/feature/seperate-artboard-rules, looks like this is already in your radar 👍

**Comment by kvendrik** [comment]
Hey @bomberstudios!

First of all thank you! I'm glad you like it! About the issue: I agree, for the `alpha` this works as in Sketch's native format an artboard is considered a layer which means you can check for it like this:

```ts
{
  layers: {
    someAwesomeRuleForArtboards({_class}) {
      if (_class === 'artboard') {
        ...
      }
    },
  }
}
```

I agree though that instinctively this doesn't make sense. If you don't know how the Sketch JSON file format works you would expect every "group" to be its own category so: page -> artboard -> group -> layer. I've started looking into how this would work [here](https://github.com/kvendrik/sketchlint/compare/feature/seperate-artboard-rules) but its still WIP and I have some other things that have higher priority for this project right now (e.g. a Sketch plugin 😄) but I'll leave this open so I can come back to this later.

Also if you like the project and wanna contribute help is always welcome. [You can find the contribution instructions here](https://github.com/kvendrik/sketchlint#-contributing).

**Comment by kvendrik** [comment]
Supported added in https://github.com/kvendrik/sketchlint/pull/2 and released as part of [`v1.0.0-alpha.5`](https://github.com/kvendrik/sketchlint/releases/tag/v1.0.0-alpha.5).
